src/cuga/backend/cuga_graph/nodes/cuga_lite/cuga_lite_graph.py

```
@@ -322,6 +325,89 @@ def _clean_empty_response_retry_meta(meta: Optional[Dict[str, Any]]) -> Dict[str
+def _normalize_checkpoint_value(value: Any) -> Any:
+    """Normalize msgpack-unsafe runtime values while preserving LangChain messages."""
+    if isinstance(value, BaseMessage):
+        return value
+
+    value_module = getattr(value.__class__, "__module__", "")
+    if value_module.startswith("numpy"):
+        if hasattr(value, "item") and callable(value.item):
+            try:
+                return _normalize_checkpoint_value(value.item())
+            except Exception:
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)